### ISSUE TITLE
Inconsistent stringification of numerics between MySQL and ClickHouse drivers

### ISSUE THREAD

**Issue by mzealey** [cube-js/cube#9050]
**Describe the bug**

When I run a query against ClickHouse, all numerics, measures etc get stringified.

When I run a query against MySQL, the measures at least don't get stringified in the response and are returned directly as JSON integers.

**Expected behavior**

Given the current docs, I would have expected all numeric responses to be stringified for all DB drivers (but actually I'd prefer it if something like https://github.com/cube-js/cube/pull/7118 were implemented)

**Version:**

Cube 1.1.9
Clickhouse 24.7
Mariadb 10.11.7

**Comment by igorlukanin** [comment]
> actually I'd prefer it if something like https://github.com/cube-js/cube/pull/7118 were implemented

Hey @mzealey 👋 It is indeed already implemented. Please see `castNumerics`: https://cube.dev/docs/reference/frontend/cubejs-client-core#loadmethodoptions

**Comment by mzealey** [comment]
Yes, however this is only on the javascript client side which we are not using.

**Comment by igorlukanin** [comment]
@mzealey Would you be so kind to elaborate what you're using?

**Comment by mzealey** [comment]
We're using Python/httpx to call the API directly. We're intending for this service to be used both from javascript but also various backend services (java, python etc). It seems to me that if the goal of cube is to standardize access to different data sources/queries, it would be great to have data types standardized on the API layer rather than the client layer?

**Comment by igorlukanin** [comment]
I don't disagree with that, sure. One more workaround you can use while it is what it is: AFAIK, numeric values are always returned as strings when the query is served from a pre-aggregation. So if you have pre-aggregations in place, the issue will be non relevant to you.
